src/screens/SwapStack/swap-finish-screen.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+import {observer} from 'mobx-react';
+
+import {SwapFinish} from '@app/components/swap';
+
+export const SwapFinishScreen = observer(() => {
```

Comment:
You should use observer only in the component, that need to react to something. If there is not references to mobx actions/observables, then you can stay with memo